Resources/Prototypes/Entities/Objects/Misc/briefcases.yml

```diff
@@ -0,0 +1,24 @@
+- type: entity
+  parent: BaseItem
+  abstract: true
+  id: BriefcaseBase
+  description: Useful for carrying items in your hands.
+  components:
+  - type: Item
+    size: 9999
```

Comment:
does it really need to be 9999, feels like it should be able to fit into backpacks with effort

Reply:
> does it really need to be 9999, feels like it should be able to fit into backpacks with effort

Is this not to prevent backpack in backpack so having a back pack full of full briefcases or is that not possible now?

Reply:
Making a briefcase fit into a backpack is definitely a solution for the lack of hands slots in the inventory. I'll change the value to 60, if you think there's a better one to use tell me.